Flutter/datagrid/scrolling.md

```diff
@@ -952,5 +952,82 @@ In the SfDataGrid, you can obtain the starting and ending indices of the visible
     );
   }
 
+{% endhighlight %}
+{% endtabs %}
+
+## Set the scroll offset on initial loading
+SfDataGrid allows you to set the scroll offset upon initial loading for both horizontal and vertical scroll bars. This can be achieved by assigning the offset value to the [initialScrollOffset](https://api.flutter.dev/flutter/widgets/ScrollController/initialScrollOffset.html) property of the [ScrollController](https://api.flutter.dev/flutter/widgets/ScrollController-class.html) for the needed vertical or horizontal controller.
+
+{% tabs %}
+{% highlight Dart %} 
+
+import 'package:syncfusion_flutter_datagrid/datagrid.dart';
+
+late EmployeeDataSource _employeeDataSource;
+late ScrollController _verticalScrollController;
+late ScrollController _horizontalScrollController;
+
+  @override
+  void initState() {
+    super.initState();
+    employees = getEmployeeData();
+    employeeDataSource = EmployeeDataSource(employeeData: employees);
+    _verticalScrollController = ScrollController(
+      initialScrollOffset: 500,
+    );
+    _horizontalScrollController = ScrollController(
+      initialScrollOffset: 600,
+    );
+  }
+
+ @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(builder: (context, constraints) {
```

Comment:
Remove layout builder widget and wrap datagrid directly.

Reply:
removed the layout builder